src/com/palmergames/bukkit/towny/utils/CombatUtil.java

```diff
@@ -138,7 +138,7 @@ public static boolean preventDamageCall(Towny plugin, TownyWorld world, Entity a
 			 * The target is in a TownBlock and...
 			 * the target is a tame wolf and we are not it's owner
 			 */
-			if ((defendingPlayer != null) || ((defenderTB != null) && ((defendingEntity instanceof Wolf) && ((Wolf) defendingEntity).isTamed() && !((Wolf) defendingEntity).getOwner().equals((AnimalTamer) attackingEntity)))) {
+			if ((defendingPlayer != null) || ((defenderTB != null) && ((defendingEntity instanceof Wolf) && ((Wolf) defendingEntity).isTamed() && !((Wolf) defendingEntity).getOwner().equals(attackingEntity)))) {
```

Comment:
Was this tested?

Reply:
The Object stays the same, the equals call does an == anyway, since OfflinePlayer does not override equals().